RawBencher/Benchers/RepoDbPersistentBencher .cs

```diff
@@ -0,0 +1,68 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using RepoDb;
+using System.Data.SqlClient;
+using RepoDb.Enumerations;
+
+namespace RawBencher.Benchers
+{
+    /// <summary>
+    /// Specific bencher for RepoDbPersistentBencher, doing no-change tracking fetch
+    /// </summary>
+    public class RepoDbPersistentBencher : BencherBase<SalesOrderHeader>
+    {
+        private DbRepository<SqlConnection> _repository;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="RepoDbPersistentBencher"/> class.
+        /// </summary>
+        public RepoDbPersistentBencher(string connectionString)
+            : base(e => e.SalesOrderId, usesChangeTracking: false, usesCaching: false)
+        {
+            _repository = new DbRepository<SqlConnection>(connectionString, ConnectionPersistency.Instance);
```

Comment:
Looking at your documentation, the directive of this is that you use the same connection throughout the lifetime of the repository. This isn't correct for this benchmark. It should be 'PerCall', as all benchers use a complete clean slate per call, as that's the realistic scenario benchmarked in this benchmark. Re-using connection objects could give advantages (like prepared queries) which others then don't have and therefore it's not allowed here. 

Please change this to connectionpersistency.PerCall.